Forge.TreeWalker/src/ITreeSchemaValidator.cs

```diff
@@ -0,0 +1,163 @@
+//-----------------------------------------------------------------------
+// <copyright file="ITreeSchemaValidator.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <summary>
+//     The ITreeSchemaValidator interface.
+// </summary>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Forge.TreeWalker
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Microsoft.Forge.DataContracts;
+    using Newtonsoft.Json.Linq;
+    using Newtonsoft.Json.Schema;
+
+    /// <summary>
+    /// [DEPRECATED]
+    /// The ITreeSchemaValidator interface defines the validation method that tests input schemas with the existing ForgeSchemaValidationRules or a customized rule from input.
+    /// </summary>
+    public interface ITreeSchemaValidator
```

Comment:
>public interface ITreeSchemaValidator [](http://example.com/codeflow?start=4&length=37)

I don't think we need an interface since we're using a static class.